README.md

```diff
@@ -20,5 +20,3 @@ Things you may want to cover:
 * Services (job queues, cache servers, search engines, etc.)
 
 * Deployment instructions
-
-* ...
```

Comment:
Deleted last line to differentiate between the README.md files on master and feature branches.